src/controllers/albums.js

```diff
@@ -0,0 +1,90 @@
+const Album = require('../models/album');
+
+exports.create = (req, res) => {
+  const album = new Album({
+    name: req.body.name,
+    year: req.body.year,
+    artist: req.params.id,
+  });
+  if (!album.artist) {
+    res.status(404).json({ error: 'The artist could not be found.' });
```

Comment:
You are repeating the same response a few times. Could be worth putting it into a helper function. 
Maybe something like:
```
const _handleArtistNotFound = (res) => {
    res.status(404).json({ error: 'The artist could not be found.' });
}
```

then in your controller:
```
if (!artist) {
    _handleArtistNotFound(res);
}
```